### PR TITLE
Legal popup fixes

### PR DESCRIPTION
**Pull request by krzkaczor**
image

### CODE REVIEW

File: app/actions/getTags.ts [hunk 1]
```diff
@@ -46,6 +56,19 @@ export const getTokenHolderAgreementTags: ThunkAction<{}, IAppState, {}> = async
   };
 };
 
+export const getReservationAgreementPlaceholders = () => {
+  return {
+    "investor-address": getPlaceholder("Your address"),
```
Comment (krzkaczor) [inline app/actions/getTags.ts]: @rudolfix this is interesting part for u

File: app/actions/getTags.ts [hunk 2]
```diff
@@ -46,6 +56,19 @@ export const getTokenHolderAgreementTags: ThunkAction<{}, IAppState, {}> = async
...
+    amount: getPlaceholder("Your money"),
```
Comment (rudolfix) [inline app/actions/getTags.ts]: Amount of funds reserved

File: app/actions/getTags.ts [hunk 3]
```diff
@@ -46,6 +56,19 @@ export const getTokenHolderAgreementTags: ThunkAction<{}, IAppState, {}> = async
...
+    "reservation-date": getPlaceholder("Today"),
```
Comment (rudolfix) [inline app/actions/getTags.ts]: Start date of reservation

File: app/actions/getTags.ts [hunk 4]
```diff
@@ -46,6 +56,19 @@ export const getTokenHolderAgreementTags: ThunkAction<{}, IAppState, {}> = async
...
+    "release-date": getPlaceholder("18 months from today"),
```
Comment (rudolfix) [inline app/actions/getTags.ts]: End date of reservation

File: app/actions/getTags.ts [hunk 5]
```diff
@@ -46,6 +56,19 @@ export const getTokenHolderAgreementTags: ThunkAction<{}, IAppState, {}> = async
...
+    "unlock-fee": getPlaceholder("10% of your funds"),
```
Comment (rudolfix) [inline app/actions/getTags.ts]: Release fee amount

File: app/actions/getTags.ts [hunk 6]
```diff
@@ -46,6 +56,19 @@ export const getTokenHolderAgreementTags: ThunkAction<{}, IAppState, {}> = async
...
+    "neumark-amount": getPlaceholder("Your reward in Neumarks"),
```
Comment (rudolfix) [inline app/actions/getTags.ts]: Neumark grant amount

File: app/actions/getTags.ts [hunk 7]
```diff
@@ -46,6 +56,19 @@ export const getTokenHolderAgreementTags: ThunkAction<{}, IAppState, {}> = async
...
+    "neumark-acquisition-ratio": getPlaceholder("Your acquisition ratio"),
```
Comment (rudolfix) [inline app/actions/getTags.ts]: Neumark acquisition ratio

File: app/components/LegalModal.tsx [hunk 8]
```diff
@@ -46,6 +46,7 @@ export const LegalModalComponent: React.SFC<InjectedFormProps & ILegalModalProps
         </div>
       </Modal.Body>
       <Modal.Footer>
+        <div className={styles.placeholder}>PLACEHOLDER</div>
```
Comment (rudolfix) [inline app/components/LegalModal.tsx]: [Text marked with orange background] will be personalized when your funds are committed and transaction is mined. You will be able to download and print your agreement.
---------------
[this should be with this orange background]

**Pull request merged**